src/cap-description.adoc

```diff
@@ -613,14 +613,21 @@ bounds are formed of two or three sections:
 * Lower bits are set to zero
 ** This is only if there is an internal exponent (EF=0)
 
-.Composition of address bounds
-[#comp_addr_bounds,options=header,align="center"]
+.Composition of the decoded top address bound
+[#comp_addr_bounds,options=header,align="center",cols="2,4,2,2"]
 |==============================================================================
-| Configuration  | Upper section | Middle Section | Lower section
-| EF=0           | address[MXLEN-1:E + MW] + ct | T[MW - 1:0] | {E{1'b0}}
-| EF=1, i.e. E=0 | address[MXLEN-1:MW] + ct   2+| T[MW - 1:0]
+| Configuration  | Upper section (if E + MW &#8804; MXLEN) | Middle Section | Lower section
```

Comment:
```suggestion
| Configuration  | Upper section (ony present if E + MW &#8804; MXLEN) | Middle Section | Lower section
```

Reply:
(I suspect that will lead to sad wrapping issues, but may not be the end of the world). Happy to add to the caption or text underneath instead/as well?

Reply:
(To be clear, I mean wrapping of the asciidoc column header, not any kind of wrapping of capability bounds!)